code/game/machinery/machinery.dm

```diff
@@ -284,6 +284,7 @@ Class Procs:
 
 //set_machine must be 0 if clicking the machinery doesn't bring up a dialog
 /obj/machinery/attack_hand(mob/user, check_power = 1, set_machine = 1)
+	..() // unbuckling etc

```

Comment:
well I'm pretty sure it's so that you don't unbuckle someone and activate a machine with one single click